blackbird/src/test/java/com/fasterxml/jackson/module/blackbird/TestNoPackageSerialization.java

```diff
@@ -0,0 +1,78 @@
+package com.fasterxml.jackson.module.blackbird;

```

Comment:
Since it's serialization test, move to `com.fasterxml.jackson.module.blackbird.ser`